carla-carma-integration/carla-carma-agent/launch/carla_carma_agent.launch

```diff
@@ -42,6 +42,23 @@
   <arg name='vehicle_width' default='3.00'/>
   <arg name='vehicle_wheelbase' default='2.79'/>
 
+  <!--
+    ###################
+    ## Virtual Vehicle Info ##
+    ###################
+  -->
+  <arg name='role_name' default='virtual_vehicle'/>
+  <arg name='vehicle_model' default='vehicle.toyota.prius'/>
```

Comment:
Is there somewhere that this (and the other vehicle parameters get linked to the CARMA parameters from VehicleConfigParams.yml, I'm not sure I see where that's happening.